resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -157,18 +203,18 @@ spec:
             runAsUser: 1000
           resources:
             {{- toYaml .Values.deployment.resources | nindent 12 }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
+      {{- end }}
       {{- with .Values.deployment.nodeSelector }}
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-    {{- if .Values.global.priorityClassName }}
```

Comment:
shouldn't we use `global.priorityClassName` instead?

Reply:
Good catch, fixed